code/modules/atmospherics/machinery/unary/cryo_cell.dm

```diff
@@ -10,7 +10,8 @@
 	var/on = 0
 	var/datum/light/light
 	var/ARCHIVED(temperature)
-	var/obj/overlay/O1 = null
+	var/image/bottom_image = null
```

Comment:
These could be removed and replaced by SafeGetOverlayImage calls

Reply:
I swapped over to SafeGetOverlayImage, but I needed to add some additional features (pixel_x and pixel_y) to the function.